server/sonar-web/src/main/js/app/components/nav/settings/SettingsNav.tsx

```diff
@@ -73,14 +73,15 @@ export default class SettingsNav extends React.PureComponent<Props> {
   renderExtension = ({ key, name }: Extension) => {
     return (
       <li key={key}>
-        <Link to={`/admin/extension/${key}`} activeClassName="active">
+        <Link activeClassName="active" to={`/admin/extension/${key}`}>
           {name}
         </Link>
       </li>
     );
   };
 
   renderConfigurationTab() {
+    const { organizationsEnabled } = this.props;
     const configurationClassNames = classNames('dropdown-toggle', {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Define a constant instead of duplicating this literal 3 times. ([typescript:S1192](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=typescript%3AS1192))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2FMMF-647%2Fadministration-console-for-webhooks&issues=AWHDO8Ba1hZr15o4gqrI&open=AWHDO8Ba1hZr15o4gqrI)

[ISSUE_KEY]: AWHDO8Ba1hZr15o4gqrI